docs/sources/installation/configuration.md

```diff
@@ -428,6 +428,102 @@ allowed_organizations = github google
 
 <hr>
 
+## [auth.gitlab]
+
+You need to [create a GitLab OAuth
+application](https://docs.gitlab.com/ce/integration/oauth_provider.html).
+Choose a descriptive *Name*, and use the following *Redirect URI*:
+
+```
+https://grafana.example.com/login/gitlab
+```
+
+where `https://grafana.example.com` is the URL you use to connect to Grafana.
+Adjust it as needed if you don't use HTTPS or if you use a different port; for
+instance, if you access Grafana at `http://203.0.113.31:3000`, you should use
+
+```
+http://203.0.113.31:3000/login/gitlab
+```
+
+Finally, select *api* as the *Scope* and submit the form. You'll get an
```

Comment:
Can be good to mention that **read_user** scope can be used instead of **api** when allowed_groups not are used.

Reply:
Thanks for the suggestion, added.